precipitation/ngsild-payloads/Ouvrage.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "urn:ngsi-ld:Device:OuvrageOPR0000144308",
+    "type": "Device",
+    "stationCode": {
+        "type": "Property",
+        "value": "OPR0000144308"
+    },
+    "commune": {
+        "type": "Property",
+        "value": "Mons"
+    },
+    "name": {
+        "type": "Property",
+        "value": "SOURCE DE LA SIAGNE"
+    },
+    "departement": {
```

Comment:
it also could be a relationship to a `department` (is it that name in EN?) entity